x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -171,6 +173,18 @@ export const ActionForm = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [connectors, actionTypesIndex]);
 
+  useEffect(() => {
+    if (connectors.length > 0) {
```

Comment:
I was testing this locally and noticed the fix doesn't work when deleting all my connectors, the save button then started becoming enabled. It worked fine after removing this line of code.

Reply:
Updated! Good catch. I had a pre-configured connector so my number of connectors was always > 0